app/views/users/login.html.erb

```diff
@@ -1,4 +1,5 @@
 <% content_for :title, _('Login') %>
+<span id="badge"></span>
```

Comment:
I don't see this id used in this PR, is this `<span>` necessary?

Reply:
@waldenraines 
There is no use for `span#badge` as you mention unless you install the `foreman_theme_satellite`.
Then it should put the RedHat logo img inside the `span#badge`.

Removing the `#badge` cause this error because patternfly use it as a placeholder:
![screenshot-2017-10-26 login](https://user-images.githubusercontent.com/1262502/32060460-77de6aa6-ba77-11e7-8379-afab41d38a6d.png)

The old version used media queries in order to override this behavior and fix this issue.
I believe we should go with the patternfly implementation and keep it as a placeholder so we have less css code.

**It means that I will need to update the `foreman_theme_satellite` project to inject the img inside the `span#badge` and not together with it before the `#login` like it does now:**
https://gitlab.sat.lab.tlv.redhat.com/satellite6/foreman_theme_satellite/blob/SATELLITE-6.3.0/app/overrides/satellite_logo_override.rb#L5-9